Change IntegerType::scale default value to 0

BenoitDuffez opened the pull request:
which is 0 and can't be changed

see symfony/symfony#26795

see https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Form/Extension/Core/DataTransformer/IntegerToLocalizedStringTransformer.php#L35: the PHPDoc says it's unused, but the doc doesn't.

BenoitDuffez committed:
set default scale value to 0

see https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Form/Extension/Core/DataTransformer/IntegerToLocalizedStringTransformer.php#L35

xabbuh commented:
This will also affect the `NumberType` where this file is used too. I would instead rather drop the inclusion of this option from the reference of the `IntegerType` instead.

BenoitDuffez commented:
Indeed. I grepped around to see if that file was included and I failed to see that it was used by `NumberType`. I will add commits to the PR.

Commit:
reverted the edit on scale.rst; removed inclusion in integer.rst

As pointed out in the comments of symfony/symfony-docs/pull/9549, the scale.rst.inc file is included twice: once in integer.rst (the goal of the PR) and once in number.rst.
number.rst should not see this edit as its scale parameter is not ignored.
Hence, the inclusion was removed from integer.rst and the edit was made.
number.rst is now identical to base ref.

BenoitDuffez commented:
Done. Note that the `scale.rst.inc` is now included only once, and could be inlined in `number.rst`.

Would that be interesting to change in this PR?

ostrolucky commented:
dupe of https://github.com/symfony/symfony-docs/pull/9542

BenoitDuffez commented:
Indeed, thanks.
